src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/reporting/generator/BookingsReportGenerator.kt

```diff
@@ -36,6 +36,7 @@ class BookingsReportGenerator : ReportGenerator<BookingsReportDataAndPersonInfo,
         isReferralEligibleForCas3 = booking.referralEligibleForCas3,
         referralEligibilityReason = booking.referralEligibilityReason,
         probationRegion = booking.probationRegion,
+        localAuthority = booking.localAuthority,
```

Comment:
Would it make sense to name it as `localAuthorityAreaName` in the report column?

Reply:
As these column names are intended to be read by the end user, I don't think `localAuthorityAreaName` communicates any additional information over just `localAuthority`, as the name (e.g. "East Midlands") is what they're going to be referring to it by.

Reply:
agree, we can keep it for report for end user.